app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential/api_create.rb

```diff
@@ -0,0 +1,36 @@
+class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential
+  module ApiCreate
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      credential = manager.with_provider_connection do |connection|
+        connection.api.credentials.create!(params)
```

Comment:
Yeah, that was the same thing I am not sure about.  Actually not just `kind`, my question is do we want to validate the input? and how much we want to validate? Or let Tower's validation does its work?